go.mod

```diff
@@ -35,10 +35,10 @@ require (
 	k8s.io/apiserver v0.27.6
 	k8s.io/client-go v0.27.6
 	k8s.io/utils v0.0.0-20230209194617-a36077c30491
-	knative.dev/eventing v0.38.1-0.20231019094926-16d75a980703
+	knative.dev/eventing v0.38.1-0.20231019170735-4d14c2126a20
 	knative.dev/hack v0.0.0-20231016131700-2c938d4918da
 	knative.dev/pkg v0.0.0-20231017113806-d6ab72900ea5
-	knative.dev/reconciler-test v0.0.0-20231019092754-317033b0f02e
+	knative.dev/reconciler-test v0.0.0-20231017131250-999d077826b7
```

Comment:
Not sure, why we are downgrading here :thinking: 
@pierDipi do you know what happens here?

Reply:
I guess it's pointing to branch 1.12 now

Reply:
Last commit is exactly https://github.com/knative-extensions/reconciler-test/commit/999d077826b726afe356c634dc6d1f5fe8185893

Reply:
so this is the correct way of aligning dependencies

Reply:
/unhold
/lgtm